proposal.rst

```diff
@@ -271,7 +312,34 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+Community Bonding Period (May 4 - May 31)
+    * Interact with mentors, introduce myself to the community.
+    * Get familiar with Xapian’s development style. Understand expected coding, documentation, and testing standards set by Xapian.
+    * Start working on some issues to familiarise with the codebase. (Not necessarily related to the project)
+    * Discuss and prepare a list of various MIME types that need to be worked on.
+    * Read documentation relating to the required libraries.
+
+First Coding Period (June 1 - June 28)
+    * Implement first handler, update build system, and add new worker to omindex. (7 days)
+    * Look for issues, take feedback, improve performance. (6 days)
+    * Document the code. (4 days)
```

Comment:
What sort of documentation are you planning? 4 days seems a long time if you are adding support for one or two file formats. If you flesh out the detail of what support you're adding in this subproject, it may become clearer what documentation you need.

Reply:
Documenting the code + Document on how to add support for other zip based xml formats. Updated the proposal to reflect this.